doc/man3/d2i_PrivateKey.pod

```diff
@@ -50,15 +50,18 @@ If the B<*a> is not NULL when calling d2i_PrivateKey() or d2i_AutoPrivateKey()
 (i.e. an existing structure is being reused) and the key format is PKCS#8
 then B<*a> will be freed and replaced on a successful call.
 
+To decode a key with type B<EVP_PKEY_EC>, d2i_PublicKey() requires B<*a> to not be
+NULL and contain an EC_KEY structure referencing the proper EC_GROUP.
```

Comment:
This is slightly confusing since the type of `*a` is not `EC_KEY` but `EVP_PKEY`. I think what you mean is that it should not be NULL and contain an EVP_PKEY structure assigned an EC_KEY referencing the proper EC_GROUP. Or something along those lines.

Reply:
Yes, that's what I'm going for. It got munged a bit with the wording that copied from.